mkosi/distributions/mageia.py

```diff
@@ -24,23 +24,31 @@ def remove_packages(cls, state: MkosiState, remove: List[str]) -> None:
 
 @complete_step("Installing Mageia…")
 def install_mageia(state: MkosiState) -> None:
+    release = state.config.release.strip("'")
+
     if state.config.local_mirror:
         release_url = f"baseurl={state.config.local_mirror}"
         updates_url = None
     elif state.config.mirror:
-        baseurl = f"{state.config.mirror}/distrib/{state.config.release}/x86_64/media/core/"
+        baseurl = f"{state.config.mirror}/distrib/{state.config.release}/{state.config.architecture}/media/core/"
```

Comment:
Did you mean to use `release` here?

Reply:
I wasn't thinking of it, but for consistency it could be used.

Reply:
Fixed with https://github.com/systemd/mkosi/pull/1282/commits/a31a639733da22abaec2322ef3c8133a673c56bb.